outputs.tf

```diff
@@ -8,6 +8,11 @@ output "cluster_id" {
   depends_on  = [null_resource.confirm_network_healthy]
 }
 
+output "cluster_account_id" {
+  description = "Account ID of cluster created"
+  value       = data.ibm_iam_account_settings.iam_account_settings.account_id
```

Comment:
Adding this not as a requested change, but a dev note:
Officially this data element would be the account of the current API Key that was supplied, which technically would be the same account as the cluster.

But the data.ibm_iam_account_settings provider does require some additional user permissions than just what is required to provision a cluster.

If you wanted to return the actual account of the cluster, and without needing extra user permissions, you could have this output use the account number from the new cluster CRN instead. That way it would be the true account number for the cluster, and not need this data element.

Reply:
Yes we need to avoid using `data.ibm_iam_account_settings` as you need to be an account admin to run it. You can get the account ID in other ways